1-Vanilla/scripts/todo_list.js

```diff
@@ -0,0 +1,130 @@
+let nextId = 0;
+
+function createButton(id, type, cb) {
+  let button = document.createElement("div");
+  let icon = "&#xe145;";
+
+  switch (type) {
```

Comment:
[optional] an alternative to that would be to create a map from "input type" to "icon", e.g:
```
const iconsMap = { "edit": "&#xe3c9", "remove": "a1#zc"...}

const icon = iconsMap[type] || "default-if-needed"
```
would make the code a bit shorter